LambdaEngine/Include/Rendering/PaintMaskRenderer.h

```diff
@@ -23,6 +23,12 @@ namespace LambdaEngine
 	class Buffer;
 	class Window;
 
+	enum class EPaintMode
+	{
+		REMOVE = 0,
+		PAINT
```

Comment:
```suggestion
		PAINT = 1
```
Since you explicitly added "= 0" to remove

Reply:
Sure, makes it easier to read